ibm/resource_ibm_is_image.go

```diff
@@ -0,0 +1,276 @@
+package ibm
+
+import (
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.ibm.com/Bluemix/riaas-go-client/clients/compute"
+	iserrors "github.ibm.com/Bluemix/riaas-go-client/errors"
+)
+
+const (
+	isImageHref            = "href"
```

Comment:
Can this be changed to isImageFile = "file"  instead of href to be inline with cli and api